### DISCUSSION TITLE
Mastodon needs to capitalize on the current twitter exodus just like we did the last time.

### DISCUSSION THREAD

**Discussion by MrLuxuri** [mastodon/mastodon#33144] [General]
We need to approach this better, brushing off the bluesky moment saying we will play the long-term game is just short-cited and lost opportunity.

We need to see where we were lagging behind (A good example: improving the mobile app, onboarding experience) and tackle them for the general average user, if we ever want this project to be more than a tech nerd's savvy platform.

If we lose everyone on Twitter/X to bluesky, despite it's disadvantages such as being propitiatory and VC-funded/focused, it will be so much harder in the future to ever get them to Mastodon, because regardless of Bluesky's disadvantages, it is quite noob friendly and isn't as bad as X/Twitter today, more than enough for the average user to settle in.

Use this thread to discuss the areas Mastodon still needs to improve and what we can do amid the 2nd X/Twitter exodus, I sure do think and hope this won't be the last final one, and this is still ongoing.

@Gargron

**Comment by ghost** [comment]
No show password option last I checked

**Comment by linduva** [comment]
It is not easy to see if your postings are attractive or not. You have no insights on how often my post is shown for example.

**Comment by linduva** [comment]
To view all answers is too complicated. Every other platform shows the answer by just clicking on the answer button, and provide a possibility to answer at the same time,
